src/navigation/navigationhandler_lua.inl

```diff
@@ -374,43 +479,80 @@ joystickAxis(std::string joystickName, int axis)
     return cmd;
 }
 
-// Directly adds to the global rotation of the camera.
+/**
+ * Directly add to the global rotation of the camera (around the focus node).
+ *
+ * \param v1 the value to add in the x-direction (a positive value rotates to the
```

Comment:
Should we rename these parameters? V1 and V2 are not very descriptive.

Reply:
I considered that, but was unsure about the naming (e.g. is it a velocity or value? ) so left it as is for now.  

Got any suggestion for alternative names?